docs/pages/reference/backends.mdx

```diff
@@ -946,7 +946,32 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
-## Athena
+## Athena (Preview)
+
+<Details
+  title="Version warning"
+  opened={true}
+  min="14.0"
+>
+  Athena audit back-end is available starting from Teleport `14.0`.
+</Details>
+
+<Admonition type="warning" title="Preview">
+  Athena audit back-end is currently in Preview.
+</Admonition>
+
+<Notice type="danger">
+
+If you are using
```

Comment:
Users may not be familiar with the `SearchEvents` API. I would use more general language, for example, referring to Teleport integrations that rely on audit events.

Does this apply to access request plugins?

Reply:
> Does this apply to access request plugins?

No.

> Users may not be familiar with the SearchEvents API.

I have added link to Search Events API.